src/projects/machinacommons_adaptor-ex.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layouts/project.liquid
+tags: ["Kultur","Theater","Games"]
+id: adaptorex
+title: MACHINA COMMONS - adaptor:ex
+abstract: adaptor:ex ist ein modulares, erweiterbares Framework um interaktive Performances, Theater-Games und andere Projekte zu erstellen
```

Comment:
@machinaeXphilip, may I suggest to also shorten this? My first review with the maximum of 140 characters was a not so thought-through estimate. If you get it down to the same character count as the "dingsda" project, it would greatly improve the visual appearance of the project cards:

![Screenshot 2022-05-19 at 10 26 24](https://user-images.githubusercontent.com/15640196/169248528-03a1c4cd-883f-4118-9889-c9280d7cc6a1.png)